dev-util/rt-tests/rt-tests-2.6.ebuild

```
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..12} )
```

```suggestion
PYTHON_COMPAT=( python3_{10..12} )
```
Please drop python3_9